physical/raft/raft.go

```diff
@@ -780,6 +807,13 @@ func (b *RaftBackend) SetupCluster(ctx context.Context, opts SetupOpts) error {
 
 	b.raft = raftObj
 	b.raftNotifyCh = raftNotifyCh
+	b.setupAutopilot(opts)
```

Comment:
Should AutoPilot be setup on follower nodes? `SetupCluster` will be called on every node.

Reply:
Followed guidance from here: https://github.com/hashicorp/raft-autopilot/blob/master/autopilot.go#L105

Reply:
Another reason was that I was thinking that when perf standby replications is enabled, follower nodes should respond to read requests. But now I wonder how can the follower nodes (who do not have a running autopilot instance), respond to state and get-config APIs.

Reply:
Can we force these APIs to be serviced by the leader?

Reply:
I was meaning to ask if its okay to add the state and config get calls to `perfStandbyAlwaysForwardPaths`.

Reply:
If yes, then probably we don't need to set up autopilot here. We can do it all in `setupRaftActiveNode`. Its just about creating autopilot instance really.

Reply:
Oh, all raft operations are already getting forwarded. So, we are good there. I'll move this to `setupRaftActiveNode`.

Reply:
Done.